tests/scripts/functional-tests-apt-ubuntu.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -ex
+
+TEST_DIRS=tests
+
+make docker_build_apt_ubuntu
+
+. .venv-jenkins/bin/activate
+
+py.test ${TEST_DIRS}/functional/ -v --test_dir=/var/jenkins_home/test_models_mzeglars
```

Comment:
avoid folder name with login name . how about /var/jenkins_home/test-ovms-models ?

Reply:
:+1: 
Beside that in future we're going to move this to Makefile anyway so this will be in a variable.
Also lets keep to the syntax and use underscores.